wasatch/applog.py

```diff
@@ -113,7 +114,7 @@ def __init__(self,
             enable_stdout=True,
             logfile=None,
             timeout_sec=5,
-            append=False):
+            append_arg="True"):
```

Comment:
Personally, I would keep the old --append boolean, and add a new argument --max-size-mb.  The new argument could apply whether or not append was in effect, and could be set to 0 to disable truncation (allow to grow unconstrained).

Reply:
> Personally, I would keep the old --append boolean, and add a new argument --max-size-mb. The new argument could apply whether or not append was in effect, and could be set to 0 to disable truncation (allow to grow unconstrained).

In the interest of time, I made a commit that still uses the True/False/Limit values for --log-append (the actual arg name is unchanged), but it is actually unlimited when set to True.

```
--log-append True # append logs do not limit
--log-append LIMIT # append logs with a max size driven by Wasatch.PY
--log-append False # clear the log file upon program restart

# all of these values are case insensitive
```

erring on the side of less configurability for the log file size because I want time to work on BLE today !